src/Dapper.FluentMap.Dommel/Dapper.FluentMap.Dommel.csproj

```
@@ -2,17 +2,21 @@
   <PropertyGroup>
     <Description>Dapper.FluentMap extension for Dommel support.</Description>
     <Copyright>Copyright © Henk Mollema 2017</Copyright>
-    <VersionPrefix>1.7.1</VersionPrefix>
+    <VersionPrefix>2.0.0</VersionPrefix>
     <Authors>Henk Mollema</Authors>
-    <TargetFrameworks>net451;netstandard1.3;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Only target .NET Standard 2.0.